### PR TITLE
Completed addition function

### PR DESCRIPTION
**Pull request by NicholasChoong**
## Change Summary
- Created an addition function

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

### CODE REVIEW

**Comment by Drewbi** [comment]
Is this in scope for the project?

**Review by scratchclaggy — APPROVED** [review]
LGTM :rocket:

**Comment by NicholasChoong** [comment]
> 

Why yeah not

File: ash/test_addition.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def add(a, b):
+    """Add a and b"""
+    return a + b
+
+
+def test_add():
+    assert add(1, 1) == 2
+    assert add(0, 0) == 0
+    assert add(1, -1) == 0
```
Comment (Drewbi) [inline ash/test_addition.py:9]: This doesn't cover the -1, -1 case

**Comment by APM246** [comment]
rubbish

**Review by Drewbi — APPROVED** [review]
lgtm

**Pull request merged**